public/css/icinga/layout.less

```diff
@@ -332,7 +332,10 @@
 
     .nav-level-1 > .nav-item i {
       font-size: 1.5em;
-      margin-right: .5em;
+
+      &:not(.nav-level-2 > .nav-item i) {
```

Comment:
This can be solved more easily. Also, both properties are only relevant to level 1. So just extend the existing selector to: 
```
.nav-level-1 > .nav-item > a > i,
.nav-level-1 > .nav-item > span > i {
```